### PR TITLE
Update additional reading and build docs

### PR DESCRIPTION
**Pull request by odjuricicTT**
As i onboard on the project adding some resources that are missing the the docs.

### CODE REVIEW

File: docs/src/build.md [hunk 1]
```diff
@@ -111,11 +111,14 @@ sudo apt install ninja-build
...
 ### Ubuntu 20.04
 
-On Ubuntu 20.04, we need to update the version because 3.20 is the minimum required for this project
+On Ubuntu 20.04 we need to instal clang and to update the version of cmake because 3.20 is the minimum required for this project.
 
 ```bash
 sudo apt-get update
 sudo apt-get upgrade
+
+sudo apt-get install clang
```
Comment (nsmithtt) [inline docs/src/build.md:120]: Is this needed? I thought ubuntu 20.04 ships with clang.

File: docs/src/build.md [hunk 2]
```diff
@@ -111,11 +111,14 @@ sudo apt install ninja-build
 
 ### Ubuntu 20.04
 
-On Ubuntu 20.04, we need to update the version because 3.20 is the minimum required for this project
+On Ubuntu 20.04 we need to instal clang and to update the version of cmake because 3.20 is the minimum required for this project.
```
Comment (nsmithtt) [inline docs/src/build.md:114]: Typo "install"

File: docs/src/build.md (hunk 1 above)
Comment (topke011) [reply docs/src/build.md:120]: I think it doesn't :(

Comment (mtopalovicTT) [reply docs/src/build.md:120]: I think it doesn't :(

**Review by nsmithtt — APPROVED** [review]
Just fix the typo please before merging, otherwise this is great, thank you!

**Pull request merged**